synapse/federation/transport/server/federation.py

```diff
@@ -797,4 +834,5 @@ async def on_GET(
     FederationRoomHierarchyUnstableServlet,
     FederationV1SendKnockServlet,
     FederationMakeKnockServlet,
+    AccountStatusServlet,
```

Comment:
Ah TIL, I was looking for something like this since it feels too inconsistent that the client-side endpoint is only enabled if the config option is turned on but the same isn't true for the federation endpoint.